metacat-main/src/test/resources/search/mapping/metacat.json

```diff
@@ -1,6 +1,7 @@
 {
     "settings": {
-        "index":{"number_of_shards":10},
+        "index":{"number_of_shards":5,
+                 "max_result_window": 2147483647},
```

Comment:
Why are we setting this property high?

Reply:
We discussed yesterday, it's needed as setting the size in the query.
 final SearchRequestBuilder request = client.prepareSearch(esIndex)
            .setTypes(type)
            .setSearchType(SearchType.QUERY_THEN_FETCH)
            .setQuery(queryBuilder)
            .setSize(Integer.MAX_VALUE);